src/main/java/org/minutebots/frc2019/Robot.java

```diff
@@ -9,8 +28,51 @@
    */
   @Override
   public void robotInit() {
+      
+  }
+
+  final Encoder leftEncoder = new Encoder(0, 0);
+  final Encoder rightEncoder = new Encoder(0, 0);
+  final DifferentialDriveOdometry m_odometry = new DifferentialDriveOdometry(new Rotation2d(0.0));
```

Comment:
I'm not sure what our plan for styling this year is (@kevintran-git?) but we may or may not use this convention during the season.